ludwig/schema/metadata/configs/llm.yaml

```diff
@@ -119,3 +119,17 @@ adapter:
     adapter_layers:
       ui_display_name: Adapter Layers
       expected_impact: 3
+quantization:
+  _oneOf:
+    allOf:
+      ui_display_name: Quantization
+      expected_impact: 3
+    none:
+      ui_display_name: No Quantization
+      expected_impact: 3
+  _meta:
```

Comment:
nice, so this will be the "top-level" metadata for the `jsonschema_type_mapping`

Reply:
Right, that's my understanding.

Reply:
So then make sure to add it to the `jsonschema_type_mapping` here:
```
{
  "oneOf": {
    ...
  },
  "title": "...",
  "description": "...",
  "parameter_metadata": ${PUT HERE}
}
```